### DISCUSSION TITLE
Does anyone tried to use Gobig?

### DISCUSSION THREAD

**Discussion by leonnn1** [hlky/stable-diffusion#234] [Ideas]
Hello I wonder if anyone was able to use GObig to upscale the images, with Gobig we would have a similar upscale possibility as in MJ detailed upscale, I found the SD version here https://github.com/jquesnelle/txt2imghd

**Comment by kwaad2** [comment]
I'm not sure if it works, I was using remote desktop to my desktop which I run the hlky version, which I updated today... It has GoBIG. I didn't get it to work though, likely the environment or something. I need to diagnose when I get home.

**Reply by leonnn1** [reply]
OH, so the newer version already has Gobig built in? I will try to update it, one question, to update it I just need to do a git pull to the directory I installed it?

**Reply by altryne** [reply]
you may need to download the goBig models as well

**Reply by leonnn1** [reply]
I think I did that, I put the file on scripts, but It didn't seem to work on the hlky, I am or I don't know how to do it (wich is probably it)
